Cogs.Tests.Integration/IntegrationTests.cs

```diff
@@ -1339,6 +1339,41 @@ public class IntegrationTests
         }
 
         [Fact]
+        public async void ReusabletoSimpleList()
```

Comment:
I added a test for a reusable that has a list of simple types only because parsing a list is different than parsing a single, so I wanted to make sure that reusable types could parse a list of a simple type.